packages/runtime/container-runtime/src/summaryCollection.ts

```diff
@@ -306,8 +308,17 @@ export class SummaryCollection {
             // from. i.e. initialSequenceNumber > summarySequenceNumber.
             // We really don't care about it for now, since it is older than
             // the one we loaded from.
-            assert(this.initialSequenceNumber > seq,
-                "Missing summary op for ack, but summary op seq > initialSequenceNumber");
+            if (seq >= this.initialSequenceNumber) {
+                // Potential causes for it to be later than our initialSequenceNumber
+                // are that the summaryOp was nacked then acked, double-acked, or
+                // the summarySequenceNumber is incorrect.
+                this.logger.sendErrorEvent({
```

Comment:
Actually -- should this be an error event?  If we recover gracefully maybe performance event instead

Reply:
Or telemetry event

Reply:
Good question. My initial understanding of our logger was that even if it isn't fatal, we should log unexpected or problem-indicative errors as errors. @vladsud in case this has changed.

Reply:
For example: summary nack logs as error even though we typically recover and move forward fine.

Reply:
I see -- seems fine to me then.

Reply:
We should log this as error.  Even though we try to be resilient, it probably indicates that there is something wrong with the server.   That is also why we keep the quorum assert, because the client can't really go on without other consequence.